odt-opt-in/initial ODT Opt-in JSON Contract.md

```diff
@@ -0,0 +1,155 @@
+   "3-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "4-This is test verbiage4. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "5-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage."
+  ],
+  "feeDisclosure": [
+   "1-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "2-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "3-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "4-This is test verbiage4. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "5-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage."
+  ],
+  "revocationInstructions": [
+   "1-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "2-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "3-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "4-This is test verbiage4. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage.",
+   "5-This is test verbiage3. This is test verbiage. This is test verbiage. This is test verbiage. This is test verbiage."
+  ]
+ }
+}
+```
+
+## PROCESSDATA
+
+UX returns updated state of each share
+```json
+{
+  "rgState": "PROCESSDATA",
+  "powerOnFileName": "BANNO.ODTOPTIN.V1.POW",
+  "userChrList": [
+    {"id": 1, "value": "0000,0010,0012,0014"},  [comma dilineated list of 4-digit share ID for those shares which are]
+    {"id": 2, "value": ""},                     [to be enrolled into ODT services. All 5 userChr lists can be used]
+    {"id": 3, "value": ""},
+    {"id": 4, "value": ""},
+    {"id": 5, "value": ""}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+
+PowerOn Response if error condition
+```json
+{
+  "memoMode": false,
+  "errorCode":"504",
+  "loggingErrorMessage": "Error Processing Update(s): [error detail]"
+}
+```
+
+PowerOn Response if successful
+```json
+{
+ "memoMode": false,
+ "results": {
+  "maxSharesExceeded": false,
+  "shareDetailUpdated": [{
+    "SID": "0000",
+    "name": "My Primary Share"
+    "balance": "######9.99"
+    "currentState": false
+   },
+   {
+    "SID": "0010",
+    "name": "SuperDuper Checking"
+    "balance": "######9.99"
+    "currentState": true
+   }
+  ]
+ }
+}
+```
+
+Error Codes (individual error codes are for ease of researching issues.
+             All error codes will be returned to the UX as '500')
+500 - MemoMode Error
```

Comment:
I don't think we need the memo mode error specifically, since we already handle it with the memoMode boolean.

Reply:
My thoughts were as follows....
Even though we report the memo mode error via JSON through the use of a boolean, because it is still an error, for the purposes of internal code and consistencies, it was tied to a specific error code. If you'd still like me to change it, just let me know .